### PR TITLE
Adds an example app for WordPressUI.

### PR DESCRIPTION
**Pull request by diegoreymendez**
### Description:

Adds an example App to WordPressUI.  It also includes a simple mechanism for extending the demos that we can offer, divided by sections.

This was copied almost verbatim from AztecExample, to avoid redoing any work.

### Details:

Make sure there are no changes to the `WordPressUI` library itself, or to any file related to the library.  The idea is to make sure this change does not affect the library at all.

### Testing:

Open the workspace.
Build the library.  Make sure it builds fine.
Build and run the main app.  Make sure it builds fine and feel free to run the fancy alert demo.

### CODE REVIEW

File: Example/Example/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
```
Comment (aerych) [inline Example/Example/AppDelegate.swift]: Can we remove these auto generated headers?

File: Example/ExampleTests/ExampleTests.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
+//
+//  ExampleTests.swift
+//  ExampleTests
```
Comment (aerych) [inline Example/ExampleTests/ExampleTests.swift]: And here.

File: Example/ExampleUITests/ExampleUITests.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
+//
+//  ExampleUITests.swift
+//  ExampleUITests
```
Comment (aerych) [inline Example/ExampleUITests/ExampleUITests.swift]: and here too. :)

**Comment by diegoreymendez** [comment]
Thanks for the review @aerych!  All issues were corrected.

**Pull request merged**